csrc/scheduler/utils.cpp

```diff
@@ -2257,7 +2257,8 @@ void propagateReshapeTransforms(Fusion* fusion, const ComputeAtMap& ca_map) {
         }
 
         bool has_reachable_loop_id = false;
-        for (auto loop_idx : c10::irange(static_cast<int64_t>(tv->getLoopDomain().size()))) {
+        for (auto loop_idx :
+             c10::irange(static_cast<int64_t>(tv->getLoopDomain().size()))) {
```

Comment:
```suggestion
             c10::irange(std::ssize(tv->getLoopDomain()))) {
```
FYI. Don't bother changing this if you are about to submit the PR.

Reply:
Will make this change in the other PR!